blog/2019/06/26/deprecating_official_rex_packages.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Deprecating official Rex packages
+author: Ferenc Erki
+tags: news
+---
+
+As part of our efforts to simplify maintenance around RexOps projects, we have decided to stop building Rex packages on our side. We recommend relying on OS packages maintained by the various distributions in their respective upstream repositories, or to install Rex from CPAN.
+
+---
+
+There are multiple reasons behind this decision, and I'd like to mention a few bigger one below.
+
+## Existing upstream packages in various distributions
+
+Many operating systems are already packaging and shipping Rex in their own respective repositories, so building packages on our own feels like a duplicate effort.
+
+We believe the OS-specific packages in general are also having much better quality than we could achieve on our own. They are maintained by their upstreams who are in much better position to follow changes of the packaging guidelines for the given distribution.
```

Comment:
`in general are also having much` -> `in general also have much` ?

Reply:
reworded as `in general are also much better in quality`